pods/authProviders/src/index.ts

```diff
@@ -43,10 +42,8 @@ export function registerProviders (
   }
 
   app.keys = [serverSecret]
-  app.use(session({}, app))
```

Comment:
I receive the following error when trying to go with token:
```
Error: Login sessions require session support. Did you forget to use `express-session` middleware?
2024-08-04 23:24:31       at vZt.logIn (/usr/src/app/bundle.js:50668:7769)
2024-08-04 23:24:31       at Yne.login.Yne.logIn (/usr/src/app/bundle.js:50668:2141)
2024-08-04 23:24:31       at m.success (/usr/src/app/bundle.js:50668:5420)
2024-08-04 23:24:31       at n (/usr/src/app/bundle.js:50668:52137)
2024-08-04 23:24:31       at /usr/src/app/bundle.js:50668:53242
```
I support this is because session/passport session

Reply:
Ok, thanks. I was not sure if it's required even if don't use the session, seems it is. Will get it back/

Reply:
Got it back.

Reply:
Works fine now